docs/containers/mysql.md

```diff
@@ -0,0 +1,25 @@
+# Mysql
+
+This is information about the mysql service :
+
+- user root : `root`
+- password root : `nextcloud`
+
+- user : `nextcloud`
+- password : `nextcloud`
+
+- database name : `nextcloud`
```

Comment:
Is this the only name? I think every variant (nextcloud, stableXX, nextcloud2 nextcloud3,...) generates its own database.